launcher/ui/pages/instance/WorldListPage.cpp

```diff
@@ -211,7 +211,7 @@ void WorldListPage::on_actionDatapacks_triggered()
         return;
     }
 
-    if(!worldSafetyNagQuestion())
+    if(!worldSafetyNagQuestion("Open World Datapacks Folder"))
```

Comment:
For translations to work, I think you need to wrap those strings in `tr()`, the one wrapping `%1` doesn't get those iirc.